### PR TITLE
Graph python functionality

### PR DESCRIPTION
**Pull request by WCastro616**
fixed node tests

### CODE REVIEW

File: rakan/cRakan/src/Node.cc [hunk 1]
```diff
@@ -14,9 +14,14 @@ namespace rakan {
...
            const uint32_t majority_pop,
-           const uint32_t minority_pop) {
+           const uint32_t minority_pop) : id_(id),
+                                          county_(county),
+                                          majority_pop_(majority_pop),
+                                          minority_pop_(minority_pop) {
+  
   neighbors_ = new unordered_set<uint32_t>;
   total_pop_ = majority_pop + minority_pop;
+  total_pop_ = minority_pop + majority_pop;
```
Comment (gooooooooooooooose) [inline rakan/cRakan/src/Node.cc]: Is `total_pop_` set twice?

**Review by gooooooooooooooose — APPROVED** [review]
Node tests passing

File: rakan/cRakan/src/Runner.cc [hunk 2]
```diff
@@ -62,8 +62,8 @@ void Runner::add_node(uint32_t node_id,
 }
 
 bool Runner::add_edge(uint32_t node_one, uint32_t node_two) {
-  if (graph_->ContainsNode(graph_->GetNode(node_one)) && 
-      graph_->ContainsNode(graph_->GetNode(node_two))) {
+  if (graph_->ContainsNode(graph_->GetNode(node_one)->id_) && 
+      graph_->ContainsNode(graph_->GetNode(node_two)->id_)) {
```
Comment (gooooooooooooooose) [inline rakan/cRakan/src/Runner.cc]: Can simplify this with just `graph_->ContainsNode(node_one)` and the same for `node_two`

Comment (WCastro616) [reply rakan/cRakan/src/Runner.cc]: oh lmao

**Review by gooooooooooooooose — APPROVED** [review]

**Pull request merged**